### ISSUE TITLE
UX Changes required

### ISSUE THREAD

**Issue by hacksdump** [supra08/about-me#1]
Clicking on text leads to no result whereas clicking elsewhere in block successfully opens the page.

**Comment by supra08** [comment]
Alright, I will look into it asap.

**Comment by Aniket21mathur** [comment]
Fixed it?

**Comment by hacksdump** [comment]
Nope, he hasn't yet. @Aniket21mathur can you somehow convince @supra08 to do so? Every little help will be appreciated.
